### ISSUE TITLE
Accessibility Compliance issue with Search button.

### ISSUE THREAD

**Issue by rtadikonda001** [AdobeDocs/pdfservices-api-documentation#580]
On Search icon
We are integrating the PDF Embedded API into our pages. We have to be Accessibility compliant , so all the icons in the page should be readable. 

While most of the icons are readable, the search icon is not readable. 
Please have a look at the attached screenshot (missing-accessibility.png). This shows that the element is missing "aria-label".
While the other icons have that label and are accessible. 

Can you please add the attribute to this Search icon?

![missing-accessibility](https://github.com/AdobeDocs/pdfservices-api-documentation/assets/109795368/6b54f859-d5a6-4b01-bf4a-64188e7ddb9d)

**Comment by cfjedimaster** [comment]
Hi, this repo is for the documentation, not support. Please post your issue here: https://community.adobe.com/t5/document-services-apis/bd-p/Document-Cloud-SDK?page=1&sort=latest_replies&filter=all
